spec/mainadditions.html

```diff
@@ -212,91 +208,167 @@ <h1>
+          1. <ins>Set _identifiers_ to SortStringListByCodeUnit(_identifiers_).</ins>
+          1. Let _result_ be a new empty List.
+          1. For each element _identifier_ of _identifiers_, do
+            1. Let _primary_ be _identifier_.
+            1. If _identifier_ is a non-primary time zone identifier in this implementation and _identifier_ is not *"UTC"*, then
+              1. Set _primary_ to the primary time zone identifier associated with _identifier_.
+              1. NOTE: An implementation may need to resolve _identifier_ iteratively to obtain the primary time zone identifier.
+            1. Let _record_ be the Time Zone Identifier Record { [[Identifier]]: _identifier_, [[PrimaryIdentifier]]: _primary_ }.
+            1. Append _record_ to _result_.
+          1. Assert: _result_ contains a Time Zone Identifier Record _r_ such that _r_.[[Identifier]] is *"UTC"* and _r_.[[PrimaryIdentifier]] is *"UTC"*.
+          1. Return _result_.
         </emu-alg>
-      </ins>
+      </emu-clause>
     </emu-clause>
-  </emu-clause>
 
-  <emu-clause id="sec-temporal-properties-of-the-legacy-date-prototype-object">
-    <h1><a href="https://tc39.es/ecma262/#sec-properties-of-the-date-prototype-object">Properties of the Date Prototype Object</a></h1>
-
-    <ins class="block">
-      <emu-clause id="sec-date.prototype.totemporalinstant">
-        <h1>Date.prototype.toTemporalInstant ( )</h1>
-        <p>This method performs the following steps when called:</p>
+    <emu-clause id="sec-temporal-date-constructor">
+      <h1><a href="https://tc39.es/ecma262/#sec-date-constructor">The Date Constructor</a></h1>
+      <emu-clause id="sec-temporal-date">
+        <h1><a href="https://tc39.es/ecma262/#sec-date">Date ( ..._values_ )</a></h1>
+        <p>This function performs the following steps when called:</p>
         <emu-alg>
-          1. Let _t_ be ? thisTimeValue(*this* value).
-          1. Let _ns_ be ? NumberToBigInt(_t_) &times; ℤ(10<sup>6</sup>).
-          1. Return ! CreateTemporalInstant(_ns_).
+          1. If NewTarget is *undefined*, then
+            1. <del>Let _now_ be the time value (UTC) identifying the current time.</del>
+            1. <ins>Let _global_ be GetGlobalObject().</ins>
+            1. <ins>Let _nowNs_ be HostSystemUTCEpochNanoseconds(_global_).</ins>
+            1. <ins>Let _now_ be 𝔽(floor(_nowNs_ / 10<sup>6</sup>)).</ins>
+            1. Return ToDateString(_now_).
+          1. Let _numberOfArgs_ be the number of elements in _values_.
+          1. If _numberOfArgs_ = 0, then
+            1. <del>Let _dv_ be the time value (UTC) identifying the current time.</del>
+            1. <ins>Let _global_ be GetGlobalObject().</ins>
+            1. <ins>Let _nowNs_ be HostSystemUTCEpochNanoseconds(_global_).</ins>
+            1. <ins>Let _dv_ be 𝔽(floor(_nowNs_ / 10<sup>6</sup>)).</ins>
```

Comment:
Oh wow, I'd really suggest reviewing this PR with GitHub's "show whitespace" option turned off so you can skip the tab level changes! None of the text you're commenting on here is changing in this PR, other than indenting it because I added a higher-level item ("Date Objects") to the table of contents. 

I'd support any of the changes you recommemend above, but this PR isn't changing any of that text so I'd prefer it to be in a separate PR to keep this one smaller. Is that OK?

Reply:
Hide whitespace FTW! 
<img width="294" alt="image" src="https://github.com/tc39/proposal-temporal/assets/277214/9e55ed0b-3b58-4bfa-9e58-0f3d7313d7e0">

Reply:
To make this PR easier to review, I split out the indenting changes into a separate commit so it's clearer that those are just cosmetic changes unrelated to this PR.

Reply:
Submitted at #2603. :+1: